mobie/image_data.py

```diff
@@ -191,3 +212,6 @@ def main():
               tmp_folder=args.tmp_folder, target=args.target, max_jobs=args.max_jobs,
               is_default_dataset=bool(args.is_default_dataset),
               transformation=transformation, unit=args.unit)
+
+if __name__ == "__main__":
```

Comment:
Please remove these two lines, this file is not meant as direct entry point.

Reply:
Yes, I put that in for easier debugging.